src/components/data/kanban/kanban.tsx

```diff
@@ -1,127 +1,72 @@
 import React, { useEffect, useState } from "react";
 
-import {
-  AttributeData,
-  DEFAULT_URL_FIELDS,
-  FieldSet,
-  field2Title,
-  formatMessage,
-  useIntl,
-} from "../../../lib";
-import {
-  GroupedAttributeDataProps,
-  getContextData,
-} from "../../../lib/data/groupedattributedata";
-import { Button, ButtonLink, ButtonLinkProps, ButtonProps } from "../../button";
+import { Button, ButtonProps } from "../../button";
 import { Select } from "../../form";
 import { Column, Grid } from "../../layout";
-import { Body, H1, H2, H3, P } from "../../typography";
+import { Body, H2, H3 } from "../../typography";
 import "./kanban.scss";
 
-export type KanbanProps = GroupedAttributeDataProps & {
-  /** If set, items are `draggable` allowing the user to rearrange them (across) columns). */
+export type KanbanProps = {
+  /** If set, items are `draggable` allowing the user to rearrange them (across columns). */
   draggable?: boolean;
-  /** The kanban "change column" (accessible) label */
+  /** The kanban-template "change column" (accessible) label */
   labelSelectColumn?: string;
 
-  /** The kanban "move object position" (accessible) label. */
+  /** The kanban-template "move object position" (accessible) label. */
```

Comment:
These are not super clear for me just from reading the description. Also, these are just props for the kanban 'data' component, so I don't think that the template should be mentioned here?